File: translations/codenet/uncommented/Go2P/starcoder/Code/s054878611.py
def solution(n, a):
    memo = [0] * (100000 + 2)
    for num in a:
        memo[num] += 1
        memo[num + 1] += 1
        memo[num + 2] += 1
    out = 0
    for num in memo:
        if num > 0:
            out = max(out, num)
    return out

def max(x, y):
    if x > y:
        return x
    return y

if __name__ == '__main__':
    n = int(raw_input())
    a = [int(i) for i in raw_input().split()]
    print solution(n, a)

File: translations/codenet/uncommented/Go2C++/starcoder/Code/s054878611.cpp
#include <iostream>
#include <vector>

using namespace std;

int solution(int n, vector<int> &a) {
        int memo[100002] = {0};
        for (int i = 0; i < n; i++) {
                memo[a[i]]++;
                memo[a[i] + 1]++;
                memo[a[i] + 2]++;
        }
        int out = 0;
        for (int i = 0; i < 100002; i++) {
                if (memo[i] > 0) {
                        out = max(out, memo[i]);
                }
        }
        return out;
}

int main() {
        int n;
        cin >> n;
        vector<int> a(n);
        for (int i = 0; i < n; i++) {
                cin >> a[i];
        }
        cout << solution(n, a) << endl;
}

